focus/operations/src/selection.rs

```diff
@@ -666,13 +696,58 @@ mod tests {
             .map(|t| t.to_string())
             .collect();
         assert_eq!(
-            project_names,
+            target_names,
             HashSet::from(["bazel://project_b/...".to_string()])
         );
 
         Ok(())
     }
 
+    #[test]
+    fn selection_remove_all() -> Result<()> {
+        let fixture = RepoPairFixture::new()?;
+        fixture.perform_clone()?;
+
+        let projects = vec![String::from("team_zissou/project_c")];
```

Comment:
That's a question for @dbernadett -- I added a new json testing file to test that multiple selected projects/targets get removed, but trying to add that to the repo caused other tests to fail. It seems like there's something internally breaking when I try to modify the testing files. Is this something work investigating more? Or does the assurance that "remove --all" passed my manual tests & is passing this simpler test suffice?

Reply:
I think this is probably sufficient. If it can find one project/target it can probably find them all. The tests probably are not written to be super robust to changes in the fixture. If you found it hard to add a new project to the fixture, lets call this good and work on the comment in my review. :)

Reply:
That fixture is probably used by a lot of other tests, which might need to change in turn if you change that fixture. I agree David on this.